incubator/virtualcluster/cmd/syncer/app/options/options.go

```diff
@@ -235,24 +236,25 @@ func getInClusterNamespace() (string, error) {
 // createClients creates a meta cluster kube client and a super master custer client from the given config and masterOverride.
 func createClients(config componentbaseconfig.ClientConnectionConfiguration, masterOverride string, timeout time.Duration) (clientset.Interface,
 	vcclient.Interface, clientset.Interface, error) {
-	if len(config.Kubeconfig) == 0 && len(masterOverride) == 0 {
-		klog.Warningf("Neither --kubeconfig nor --master was specified. Using in-cluster API client.")
+	if len(config.Kubeconfig) == 0 {
+		klog.Warningf("--super-master-kubeconfig was not specified. Using in-cluster API client.")
```

Comment:
the condition of this warning should be the same as the follow one.
or just delete it?